Respect image ratio and tweak the responsiveness

tonysm opened the pull request:
### Changed

- Respeitando o ratio das imagens
- Alguns tweaks no responsive da home

---

closes #39 
closes #69 

**Home**
![image](https://user-images.githubusercontent.com/1178621/96386616-abba7680-1172-11eb-895f-90692e8bd268.png)

**Página do Candidato**
![image](https://user-images.githubusercontent.com/1178621/96386624-be34b000-1172-11eb-974c-61218acdad8a.png)

anapaulagomes committed:
Atualiza termos de uso e refatora método para carregar conteúdo de arquivo

anapaulagomes committed:
Adiciona termos de uso como template

tonysm committed:
Respect image ratio and tweak the responsiveness

tonysm committed:
Tweaks on the image width

tonysm commented:
**Atenção:** Ainda em discussão na issue

danielfireman commented:
> **Atenção:** Ainda em discussão na issue

Está ficando massa! Marquei como draft para deixar claro que não devo revisar.

Me marca quando chegar a hora da revisão, por favor.

anapaulagomes committed:
Add legalCode

anapaulagomes committed:
Atualiza versão do descriptor

anapaulagomes committed:
Só mostra o termo se o candidato ainda não aceitou

anapaulagomes committed:
Mostra mês em português

danielfireman committed:
Merge pull request #47 from anapaulagomes/add-terms-of-use

Atualiza termos de uso

tonysm committed:
Merge branch 'master' into square-pictures

* master:
  Mostra mês em português
  Só mostra o termo se o candidato ainda não aceitou
  Atualiza versão do descriptor
  Add legalCode
  Adiciona termos de uso como template
  Atualiza termos de uso e refatora método para carregar conteúdo de arquivo
  Adiciona google analytics
  Corrigindo nome de variável de ambiente
  Corrigindo script de deployment
  Passando FALE_CONOSCO_EMAIL
  Fix not printing the biography
  Back to padding-top 4 instead of 5
  Increase padding
  Add empty states
  Remove HTML tags from error and success message
  Terminando a funcionalidade fale_conosco
  Corrige texto dos cards e do perfil dos candidatos
  Corrige cálculo da métrica de transparência
  Passando o número máximo de pautas via servidor

tonysm committed:
Implement design changes in the card layout

- Increase transparency text size (and also the image)
- Make tag pills design bold and allow them to wrap (take another like)
- Group name and city separately than the position and number

tonysm committed:
Implement design changes in the candidate page

- Group name and city separately than position and ballot number
- Tweak the contact spacing and position

tonysm committed:
Adds font-awesome and an acknowledgements section on the readme

tonysm committed:
Only show the contact icons in the candidate page
